### ISSUE TITLE
I get "line not understood" error

### ISSUE THREAD

**Issue by rmohr** [release-engineering/backvendor#70]
When running `backvendor` from master against `kubevirt.io/kubevirt` I get the following error:

```
2018/09/21 17:39:32 /home/rmohr/go/src/kubevirt.io/kubevirt: line not understood: 100644 blob 483f8ffa1669545951f89ccdc5372013edc05771	vendor/github.com/russross/blackfriday/testdata/Amps and angle encoding.html
```

That did not happen two days ago.

**Comment by twaugh** [comment]
Should be fixed by #69.

**Comment by rmohr** [comment]
I can confirm that with #69 applied, the issue is gone.

**Comment by twaugh** [comment]
Fixed via #69.
